components/StoryLinks/StoryShareUrl.module.scss

```diff
@@ -27,10 +27,7 @@
     position: absolute;
     transition: opacity 0.4s ease;
     will-change: opacity;
-    left: -9rem;
-    right: 0;
-    top: 0;
-    bottom: 0;
+    inset: 0 0 0 -9rem;
```

Comment:
This is a new rule in the standard stylelint config, and it might need getting used to, but it's a bit cleaner with this notation.